Production/govuk_ios/Extensions/Analytics/AppEvent+Convenience.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+import Foundation
+
+extension AppEvent {
+    static var appLoaded: AppEvent {
+        let device = UIDevice.current
+        return .init(
+            name: "app_loaded",
+            params: [
+                "device_name": device.name,
+                "device_model": device.model,
+                "system_name": device.systemName,
+                "system_version": device.systemVersion
```

Comment:
Should we be adding actual device model rather than the `UIDevice.model` string which is quite generic?

It requires a bit of work to do, but would give us specific device model identifiers.

There's an example here which would probably make sense to use (DRY etc):
https://github.com/govuk-one-login/mobile-ios-networking/blob/main/Sources/Networking/HeaderTypes/DeviceModel.swift

Reply:
Discussed offline and pushed up Robin's suggestion. UIDevice.name previously returned the full model name, but seemingly only for the simulated app. See the commit for more detail: https://github.com/alphagov/govuk-mobile-ios-app/pull/26/commits/e1a79af96d333b2c72fdfa5771b44616d916b0c7.

Reply:
I note that the device model property doesn't seem to be being set on GA. It would be good to get that working?

Reply:
Summary of the issue:

The custom parameters we're sending to GA are being received, including `device_model` which is accurate for real devices (not simulator). However, the default device model filter in Firebase is not being set. Device information is normally collected by default without the need of custom logging, however Andrew Bruce (PA) concluded that this isn't currently possible on GOV.UK, due to the user agent not being sent to GA and we should stick to the custom device model logging in this PR.